rubygem-foreman_host_extra_validator/foreman_host_extra_validator.spec

```diff
@@ -0,0 +1,82 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_host_extra_validator
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+
+Summary:    Foreman plugin to add extra validations to a host.
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    0.0.4
+Release:    2%{?foremandist}%{?dist}
+Group:      Applications/System
+License:    GPLv3
+URL:        https://github.com/FILIADATAGmbH/foreman_host_extra_validator
+Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires:   foreman >= 1.11.0
```

Comment:
Note: update engine.rb to specify this if that's what the plugin needs, else I think just say `foreman`